spec/factories.rb

```diff
@@ -0,0 +1,21 @@
+FactoryGirl.define do
+
+  factory :user do |u|
+    u.email { 'example@yahoo.com' }
```

Comment:
Consider using a sequence to generate emails, so that users created through FactoryGirl are unique.